ES9/rest & spread for objects/rest_spread_example.js

```diff
@@ -0,0 +1,69 @@
+const user = {
+    firstName: "Nagasai",
+    lastName: "P",
+    session: "Javascript",
+    city: "Hyderabad",
+    state: "telangana"
+}
+
+//how to create a new object without city property
+
+const {...newuser} = user;
+console.log(newuser);
+delete newuser.city
```

Comment:
this is not the way should do it.

const {city, ...restOfUser} = user;
here restOfUser will contain everything except city